docs/dev/build_riscv64.md

```diff
@@ -0,0 +1,184 @@
+   ```
+
+3. Create a build folder:
+   ```sh
+   mkdir build && cd build
+   ``` 
+
+4. To cross compile OpenVINO Runtime for RISC-V devices, run `cmake` with specified `CMAKE_TOOLCHAIN_FILE` and `RISCV_TOOLCHAIN_ROOT`:
+   ```sh
+   cmake .. \
+     -DCMAKE_BUILD_TYPE=Release \
+     -DCMAKE_INSTALL_PREFIX=<openvino_install_path> \
+     -DENABLE_INTEL_CPU=ON \
+     -DENABLE_INTEL_GPU=OFF \
+     -DENABLE_INTEL_NPU=OFF \
+     -DENABLE_AUTO=ON \
+     -DENABLE_HETERO=ON \
+     -DENABLE_MULTI=ON \
+     -DENABLE_JS=OFF \
+     -DCMAKE_TOOLCHAIN_FILE=../cmake/toolchains/riscv64-071-thead-gnu.toolchain.cmake \
+     -DRISCV_TOOLCHAIN_ROOT=/opt/riscv \
+     -DENABLE_TESTS=ON \
+     -DENABLE_SAMPLES=ON
+   ```
+   > **NOTE**: The `riscv-gnu-toolchain` is build as there are essential files used for cross compilation under `/opt/riscv/sysroot`. The latest stable versions of Clang or GCC both support compiling source code into RISC-V instructions, so it is acceptable to choose your preferable compilers by specifying `-DCMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER`. But remember to add the key `-DCMAKE_SYSROOT=/opt/riscv/sysroot`, otherwise many fundamental headers and libs could not be found during cross compilation. 
+
+   Then run `make` to build the project:
+   ```sh
+   make install -j$(nproc)
+   ```
+
+### Build the OpenVINO Runtime Python API
+
+To enable cross-compilation with python, the library `libpython3-dev:riscv64` should be on the host machine.
+
+When installing packages using the utilities `apt` or `apt-get` the packages are downloaded from apt software repositories. On Ubuntu the apt software repositories are defined in the `/etc/apt/sources.list` file or in separate files under the `/etc/apt/sources.list.d/` directory. Host machine contains host-specific repositories (for example, x86-x64) in these files. 
+
+1. Add riscv64 repositories to download riscv64-specific packages:
+    ```sh
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy main >> riscv64-sources.list
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy universe >> riscv64-sources.list
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-updates main >> riscv64-sources.list
+    echo deb [arch=riscv64] http://ports.ubuntu.com/ubuntu-ports/ jammy-security main >> riscv64-sources.list
+    mv riscv64-sources.list /etc/apt/sources.list.d/
+    dpkg --add-architecture riscv64
+    apt-get update -o Dir::Etc::sourcelist=/etc/apt/sources.list.d/riscv64-sources.list
+    ```
+
+2. Install `libpython3-dev:riscv64` using `apt-get`:
+    ```sh
+    apt-get install -y --no-install-recommends libpython3-dev:riscv64
+    ```
+   Create  symbolink to allow python to find `riscv64-linux-gnu/python3.10/pyconfig.h` in `/usr/include/python3.10/` (this header is initially stored in `/usr/include/riscv64-linux-gnu/`)
+    ```sh
+    ln -s /usr/include/riscv64-linux-gnu/ /usr/include/python3.10/
+    ```
```

Comment:
What is about to move this part to `Software Requirements`? There is nothing about `libpython3-dev:riscv64` in that section 🤔 
What do you think?

Reply:
I think maybe leave this part here might be better. This part is actually optional and not *required*, so when you want to enable python API you could jump directly to this part and go through the whole process.